iotedgehubdev/utils.py

```diff
@@ -196,3 +196,11 @@ def get_sha256_hash(val):
         hash_object = sha256(val.encode('utf-8'))
 
         return str(hash_object.hexdigest()).lower()
+
+    @staticmethod
+    def get_device_ca_file_paths(root_dir, cert_id):
+        result = {}
+        result[cert_id + EC.CERT_SUFFIX] = os.path.join(root_dir, cert_id + EC.CERT_SUFFIX)
```

Comment:
how about remove the cert_id from the result key? only use fixed keys. Thus when calling this method there is not need to calculate the key, instead it just use the const to pick the value